docs/user-guide/advanced-usage.md

```diff
@@ -0,0 +1,143 @@
+# Advanced Usage
+
+## Asynchronous Operations
+
+You can opt-in to an asynchronous interface via the asynchronous keyword argument for methods that kick-off Tamr operations.
+
+E.g.:
+
+```python
+operation = project.unified_dataset().refresh(asynchronous=True)
+# do asynchronous stuff while operation is running
+operation.wait() # hangs until operation finishes
```

Comment:
There's a bug here - `Operation.wait()` returns the "resolved operation". This line runs `operation.wait()` and throws away the result; the next line does `op.succeeded()` but `op` is not defined, and if replaced with `operation` it will fail because it will inspect the pre-`wait()` operation, not the post-`wait()` operation.
So this should be:
```python
operation = project.unified_dataset().refresh(asynchronous=True)
# do asynchronous stuff while operation is running
op = operation.wait() # hangs until operation finishes
assert op.succeeded()
```